recipe/meta.yaml

```diff
@@ -90,7 +90,8 @@ test:
     - ax.utils.tutorials
   commands:
     - pip check
-    - pytest -vvv --capture=tee-sys --pyargs ax
+    # Skip test suite for now
```

Comment:
Please just delete this. It is really confusing to leave things lingering. Feel free to open an issue. 

Also remove `pytest` as a requirement for the test phase.

Then i can add automerge.

Reply:
I added myself as a maintainer as I plan to figure out the test suite later; if that's okay. Please don't add automerge for now.